### ISSUE TITLE
Move readMatrix methods from PoN class to HDF5File in hdf5-java-bindings.

### ISSUE THREAD

**Issue by droazen** [broadinstitute/gatk#2876]
@samuelklee commented on [Sun Aug 07 2016](https://github.com/broadinstitute/gatk-protected/issues/637)

I moved these into HDF5File in my branch, but HDF5File was moved to hdf5-java-bindings before I could merge.

**Comment by samuelklee** [comment]
@lbergelson We should do some cleanup of the HDF5 code at some point.  See also #3763.

**Comment by lbergelson** [comment]
@samuelklee I agree, it just hasn't been a priority ever so I've never gotten to it.
